packages/grid/src/vaadin-grid.js

```diff
@@ -535,42 +535,73 @@ class Grid extends ElementMixin(
     }
   }
 
+  __getIntrinsicWidth(col) {
+    const initialWidth = col.width;
+    const initialFlexGrow = col.flexGrow;
+
+    col.width = 'auto';
+    col.flexGrow = 0;
```

Comment:
For some reason commenting this out doesn't break a single test

Reply:
added a test.
done